lib/committee/middleware/request_validation.rb

```diff
@@ -58,6 +59,15 @@ def handle(request)
       if link
         validator = Committee::RequestValidator.new(link, check_content_type: @check_content_type)
         validator.call(request, request.env[@params_key])
+
+        if @coerce_date_times
+          request.env["rack.request.query_hash"].merge!(
+              Committee::ParameterConverter
+                  .new(request.env["rack.request.query_hash"], link.schema)
```

Comment:
Do you think we could retool this interface slightly?

`ParameterConverter` is quite a generally named class that seems like it could be doing work beyond just coercing datetimes. Instead of checking for `coerce_date_times` here, can we pass that into this class' constructor and have it make that decision? Like this:

``` ruby
request.env["rack.request.query_hash"].merge!(
    Committee::ParameterConverter
        .new(request.env["rack.request.query_hash"], link.schema,
          :coerce_date_times => @coerce_date_times)
```

Thoughts?